sequence_generator_main.cpp

```diff
@@ -25,26 +25,45 @@ DEFINE_uint32(N, 200, "Short hand for --num_seqs");
 DEFINE_uint32(seq_len, 256, "The length of sequence to be generated");
 DEFINE_uint32(L, 256, "Short hand for --seq_len");
 
-DEFINE_double(mutation_rate, 0.015, "Rate of point mutation rate for sequence generation");
-DEFINE_double(r, 0.015, "Short hand for --mutation_rate");
+DEFINE_uint32(group_size, 2, "The of sequences in each group");
+DEFINE_uint32(G, 2, "Short hand for --group_size");
 
-DEFINE_double(block_mutation_rate, 0.02, "The probability of having a block permutation");
-DEFINE_double(R, 0.02, "Short hand for --block_mutation_rate");
+DEFINE_double(mutation_rate, 0.3, "Maximum rate of point mutation for sequence generation");
```

Comment:
Hmm, repeating all these flags is a bit ugly. But Gflags style guide does have something like 'avoid flags in libraries'.